src/dom-to-image-more.js

```diff
@@ -122,6 +147,34 @@
                 return clone;
             });
         }
+
+        function makeSvgDataUri(node) {
+            let width = options.width || util.width(node);
+            let height = options.height || util.height(node);
+
+            return Promise.resolve(node)
+                .then(function (svg) {
+                    svg.setAttribute('xmlns', 'http://www.w3.org/1999/xhtml');
+                    return new XMLSerializer().serializeToString(svg);
+                })
+                .then(util.escapeXhtml)
+                .then(function (xhtml) {
+                    const foreignObjectSizing =
+                        (util.isDimensionMissing(width)
+                            ? ' width="100%"'
```

Comment:
Restored the `100%` when we have no dimensions at all so the `foreignObject` take up the entire `svg`'s space... handles when either or both height and width are not set in any way.